### PR TITLE
add ktlint plugin to project and apply default code style

### LINKED ISSUE #2185: Use ktlint for common code formatting among all contributors
I think it would be a good idea, it would make reviewing big PRs like #2175 easier because there will be no reformatting in the diff.

cc @charlag

### PR DESCRIPTION
**Pull request by connyduck**
closes #2185

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/AboutActivity.kt [hunk 1]
```diff
@@ -32,7 +32,7 @@ class AboutActivity : BottomSheetActivity(), Injectable {
 
         binding.versionTextView.text = getString(R.string.about_app_version, getString(R.string.app_name), BuildConfig.VERSION_NAME)
 
-        if(BuildConfig.CUSTOM_INSTANCE.isBlank()) {
+        if (BuildConfig.CUSTOM_INSTANCE.isBlank()) {
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/AboutActivity.kt:35]: 😢

**Comment by connyduck** [comment]
I will integrate the formatting check into the CI pipeline once this is merged

**Comment by charlag** [comment]
What are the rules and when does it kick in? Can we integrate it with .editorconfig or somehow else with IDE?

**Comment by connyduck** [comment]
The plugin adds a task `ktlintApplyToIdea` that you run once and then it is set up. The rules are documented here: https://github.com/pinterest/ktlint

**Pull request merged**